src/stores/game.js

```diff
@@ -491,6 +493,8 @@ export const useGameStore = defineStore('game', {
         io.socket.post(`/api/game/${this.id}/ready`, (res, jwres) => {
           if (jwres.statusCode === 200) {
             return resolve(res);
+          } else if (jwres.statusCode === 400 && res.code === 'ALREADY_STARTED') {
+            return resolve(this.id);
```

Comment:
I don't love this way of propagating the specific error through to the component from the server, but I'm not sure what would be better. Basically when the game is already started, we want a specific handler that navigates you to the game automatically